packages/resolver/src/translators/reactjs/hooks/useResolverSetShowNotice.tsx

```diff
@@ -1,21 +1,24 @@
 import {UseMutationOptions} from '@tanstack/react-query'
 
+import {useMutationWithInvalidations} from '@yoroi/common'
+
 import {useResolver} from '../provider/ResolverProvider'
-import {useMutationWithInvalidations} from '../../../utils/useMutationWithInvalidations'
 
 export const useResolverSetShowNotice = (
   options?: UseMutationOptions<void, Error, boolean>,
 ) => {
   const {showNotice} = useResolver()
 
   const mutation = useMutationWithInvalidations({
+    mutationFn: async (value: boolean) => {
+      await showNotice.save(value)
+    },
+    invalidateQueries: [['resolver', 'show-notice']],
```

Comment:
some hooks we keep the hook itself
```suggestion
    invalidateQueries: [[useResolverSetShowNotice']],
```